plugins/server/dev.hayden/khealth/versions.ktor.yaml

```
@@ -0,0 +1,6 @@
+# This file maps ktor versions to the required artifacts
+# See templates/versions.ktor.yaml for more details
+"[2.0,)": dev.hayden:khealth:2.1.+
+"[2.1,)": dev.hayden:khealth:2.1.+
+"[2.2,)": dev.hayden:khealth:2.1.+
+"[3.0,)": dev.hayden:khealth:3.0.+
```

Looks like this should just be:

```suggestion
"[2.0,)": dev.hayden:khealth:2.1.+
"[3.0,)": dev.hayden:khealth:3.0.+
```